storybook/src/date/Date.stories.tsx

```diff
@@ -35,6 +35,10 @@ export default {
   },
 };
 
-const date = "2020-02-25" as unknown as Date;
```

Comment:
this was terrible example which should even work. Usage should be `<Date>{new Date(date)}</Date>`, and in your new example
```
 <Date format="MM/dd/yyyy">
   {new Date("2020-02-25")}
</Date>